src/components/organisms/Form/index.tsx

```diff
@@ -1,43 +1,27 @@
-import React, { FC } from 'react';
-
-import { IUseFormProps, useForm, FormContext } from './hooks';
 import FormButton from './FormButton/FormButton';
 import FormTextField from './FormTextField/FormTextField';
 import FormCheckboxField from './FormCheckboxField/FormCheckboxField';
 import FormCheckboxGroupField from './FormCheckboxGroupField/FormCheckboxGroupField';
 import FormRadioGroupField from './FormRadioGroupField/FormRadioGroupField';
 import FormDropdownField from './FormDropdownField/FormDropdownField';
 import FormDropdownFilteredField from './FormDropdownFilteredField/FormDropdownFilteredField';
-import FormComponent from './Form/Form';
 
 interface IFormStatic {
   Button: typeof FormButton;
   CheckboxField: typeof FormCheckboxField;
   CheckboxGroupField: typeof FormCheckboxGroupField;
   DropdownField: typeof FormDropdownField;
   DropdownFilteredField: typeof FormDropdownFilteredField;
-  Form: typeof FormComponent;
   RadioGroupField: typeof FormRadioGroupField;
   TextField: typeof FormTextField;
 }
 
-export const Form: IFormStatic & FC<IUseFormProps> = ({ initialValues, onSubmit, validate, onChange, ...rest }) => {
-  const context = useForm({
-    initialValues,
-    onSubmit,
-    validate,
-    onChange,
-  });
-  return <FormContext.Provider value={context} {...rest} />;
+export const Form: IFormStatic = {
+  Button: FormButton,
+  CheckboxField: FormCheckboxField,
+  CheckboxGroupField: FormCheckboxGroupField,
+  DropdownField: FormDropdownField,
+  DropdownFilteredField: FormDropdownFilteredField,
+  RadioGroupField: FormRadioGroupField,
+  TextField: FormTextField,
 };
```

Comment:
Should we explicitly document that you need `Formik` and `FormikForm` to make them work? ✍️

Reply:
For sure. Gonna add it, I just wanted to get everyone's opinion on the changes first

Reply:
docs added